keyboards/alico48/info.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
This file is incorrect:
![image](https://user-images.githubusercontent.com/4781841/132934832-6fbdae68-306f-47ff-8e98-457067362e4a.png)

Reply:
Still not correct. Please use the Configurator to check the positioning and ordering of the info.json layout matches the LAYOUT macro, by pressing Ctrl+Shift+I and turning on Fast Input. When you generate the info.json, don't use rotation either.